distro/configs/superset/dashboard-assets/databases/PostgreSQL.yaml

```diff
@@ -0,0 +1,16 @@
+database_name: PostgreSQL
```

Comment:
@enyachoke  this database should probably be named "Analytics", isn't it?
and should we update the file name to reflect that as well?